src/metatracer.rs

```diff
@@ -30,20 +30,60 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-use std::sync::atomic::{AtomicU32, Ordering};
+use std::{
```

Comment:
I'm not convinced this is easier to read than just:

```rust
use std::cell::RefCell;
use std::sync::atomic::...
```

and so on. In fact, I think it's mostly harder to read (especially the triple nested `AtomicU32, Ordering` bit!).

Reply:
Just trying it out tbh. It was new in Rust recently. We can go back to the old way. No worries.